cli/bin/cmdline.rs

```diff
@@ -2,3 +2,28 @@
 // Copyright Open Network Fabric Authors
 
 //! Cmd line to start cli
+
+use clap::Parser;
+
+pub(crate) const DEFAULT_DATAPLANE_PATH: &str = "/var/run/dataplane/cli.sock";
+pub(crate) const DEFAULT_CLI_BIND: &str = "/var/run/dataplane/cliclient.sock";
+
+#[derive(Parser)]
+#[command(about = "Hedgehog Fabric Gateway CLI", long_about = None)]
```

Comment:
Still not a fan of naming Hedgehog in the project :confused:.

And you're adding yet a new variant here. Grepping in the repo, I find:
- Hedgehog Dataplane
- Hedgehog Open Network Fabric
- Hedgehog's Open Network Fabric
- Hedgehog Open Fabric Network   :upside_down_face:
- Hedgehog EVPN router           :thinking:
- Hedgehog Gateway               <-- Should we at least remove “Fabric” and use this instead?

Reply:
I'd be happy to revive this debate. The internal doc I did about OSS is nearly one-year old, and not much has changed since then (this doc contains a number of names proposals if you're interested :see_no_evil:). I suppose that with our current release target, it's still not our priority :disappointed: